src/common/components/issue-filing-button.tsx

```diff
@@ -42,27 +42,27 @@ export class IssueFilingButton extends React.Component<IssueFilingButtonProps, I
 
     public render(): JSX.Element {
         const { issueDetailsData, userConfigurationStoreData, deps } = this.props;
-        const { environmentInfoProvider, bugFilingServiceProvider } = deps;
+        const { environmentInfoProvider, issueFilingServiceProvider } = deps;
         const envInfo: EnvironmentInfo = environmentInfoProvider.getEnvironmentInfo();
-        const selectedBugFilingService: BugFilingService = bugFilingServiceProvider.forKey(userConfigurationStoreData.bugService);
-        const selectedBugFilingServiceData: BugServiceProperties = selectedBugFilingService.getSettingsFromStoreData(
+        const selectedIssueFilingService: IssueFilingService = issueFilingServiceProvider.forKey(userConfigurationStoreData.bugService);
+        const selectedIssueFilingServiceData: IssueFilingServiceProperties = selectedIssueFilingService.getSettingsFromStoreData(
             userConfigurationStoreData.bugServicePropertiesMap,
         );
-        const isSettingValid = selectedBugFilingService.isSettingsValid(selectedBugFilingServiceData);
+        const isSettingValid = selectedIssueFilingService.isSettingsValid(selectedIssueFilingServiceData);
         const href = isSettingValid
-            ? selectedBugFilingService.issueFilingUrlProvider(selectedBugFilingServiceData, issueDetailsData, envInfo)
+            ? selectedIssueFilingService.issueFilingUrlProvider(selectedIssueFilingServiceData, issueDetailsData, envInfo)
             : null;
         const target: string = isSettingValid ? '_blank' : '_self';
 
         const needsSettingsContentProps: IssueFilingNeedsSettingsContentProps = {
             deps,
             isOpen: this.state.showNeedsSettingsContent,
-            selectedBugFilingService,
-            selectedBugData: issueDetailsData,
-            selectedBugFilingServiceData,
+            selectedIssueFilingService,
+            selectedIssueData: issueDetailsData,
+            selectedIssueFilingServiceData,
             onClose: this.closeNeedsSettingsContent,
-            bugFileTelemetryCallback: this.trackFileIssueClick,
-            bugServicePropertiesMap: userConfigurationStoreData.bugServicePropertiesMap,
+            fileIssueTelemetryCallback: this.trackFileIssueClick,
+            issueFilingServicePropertiesMap: userConfigurationStoreData.bugServicePropertiesMap,
         };
         const NeedsSettingsContent = this.props.needsSettingsContentRenderer;
 
```

Comment:
Rename `create-bug-button` style below (line 72) to `file-issue-button`?

Reply:
Also rename `create-bug-button-help` in issue-filing-needs-settings-help-text.tsx?